app/src/Command/BackupRestoreRunCommand.php

```
@@ -0,0 +1,79 @@
+    {
+        $token = (string)$args->getArgument('token');
+        $restoreId = (string)($args->getArgument('restore_id') ?? '');
+
+        try {
+            (new BackupRestoreRunnerService())->run(
+                $token,
+                $restoreId,
+                fn(string $message): ?int => $io->out($message),
+                function () use ($io): void {
+                    $exitCode = $this->executeCommand(UpdateDatabaseCommand::class, [], $io);
+                    if ($exitCode !== null && $exitCode !== self::CODE_SUCCESS) {
```

PHPStan (2.2.2)

[error] Access to undefined constant App\Command\BackupRestoreRunCommand::CODE_SUCCESS.


(classConstant.notFound)